gtk4/src/print_job.rs

```diff
@@ -8,22 +8,24 @@ use crate::PrintJob;
 
 impl PrintJob {
     #[doc(alias = "gtk_print_job_send")]
-    pub fn send<P: Fn(&PrintJob, &glib::Error) + 'static>(&self, callback: P) {
+    pub fn send<P: Fn(&PrintJob, Result<(), glib::Error>) + 'static>(&self, callback: P) {
         let callback_data: Box_<P> = Box_::new(callback);
-        unsafe extern "C" fn callback_func<P: Fn(&PrintJob, &glib::Error) + 'static>(
+        unsafe extern "C" fn callback_func<P: Fn(&PrintJob, Result<(), glib::Error>) + 'static>(
             print_job: *mut ffi::GtkPrintJob,
             user_data: glib::ffi::gpointer,
             error: *const glib::ffi::GError,
         ) {
             let print_job = from_glib_borrow(print_job);
-            let error = from_glib_borrow(error);
+            let result = if error.is_null() {
+                Ok(())
+            } else {
+                Err(from_glib_none(error))
```

Comment:
This should be 
```suggestion
                Err(from_glib_full(error))
```

Reply:
https://docs.gtk.org/gtk4/callback.PrintJobCompleteFunc.html

says that the error is owned by the caller of the callback, so I do not think we should do `from_glib_full`.
Note that the old code was doing `from_glib_borrow`, so we could keep that and pass `Result<(), &glib::Error>`, but `Result<(), glib::Error>` seems more consistent with other APIs in gtk4-rs